samples/bot-formatting-cards/csharp/README.md

```diff
@@ -125,6 +125,47 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
             }
             ]
         ```
+   
+   **Note: In Adaptive Card what we are defining (User details) should be exist in the same tenant where you are testing the app (teams login) etc...**
+  - Locate the Basic info section on the user's Profile page. The Object ID that is displayed is the user's unique object ID.
```

Comment:
The Object ID that is displayed is the user's unique object ID.

User-Object-ID

Reply:
Fixed